client/components/drop-zone/index.jsx

```diff
@@ -144,6 +153,25 @@ export const DropZone = React.createClass( {
 			// from tracked nodes since another "real" event will be triggered.
 			this.dragEnterNodes = without( this.dragEnterNodes, window );
 		}
+
+		this.toggleDropZoneReduxState( this.state.isDraggingOverDocument || this.state.isDraggingOverElement );
+	},
+
+	toggleDropZoneReduxState( isVisible ) {
+		/**
+		 * The DropZone component is included in components/libraries which are not yet reduxified,
+		 * for which `connect` ( from `react-redux` ) complains that it can't find `store`
+		 * in the current context.
+		 *
+		 * For places where the reduxification has happened, this will work.
+		 */
+		if ( this.context.store ) {
+			if ( isVisible ) {
+				this.context.store.dispatch( showDropZone() );
+			} else {
+				this.context.store.dispatch( hideDropZone() );
+			}
+		}
```

Comment:
As per our private conversation on Slack yesterday, the problem I'm trying to solve here is not about testing.

The `DropZone` component is being included in places where Redux still hasn't been introduced at all or has only partial support. 

This case tries to handle `DropZone` being included from TinyMCE, where the Redux store is missing and `connect` complains that it can't find `store` in the current context. 

Will try another solution that will work better without reinventing the wheel.